pages/api/getAudio.ts

```diff
@@ -0,0 +1,27 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { groq } from 'next-sanity';
+import { sanityClient } from '@/sanity';
+import { Audio } from '@/typings';
+
+const query = groq`
+    *[_type == "audio"]
+`;
+type Data = {
+  audio: Audio[];
+};
+
+export const getAudioStaticProps = async () => {
+  return await sanityClient.fetch<Audio[]>(query);
+};
```

Comment:
Added this reusable fn that gets used below in the handler and when you need to getStatic props.

Use these functions rather than your fetch functions. (you cannot hit localhost:3000 when your server is started yet... aka during the build process.)